contractcourt/htlc_outgoing_contest_resolver.go

```diff
@@ -58,35 +58,32 @@ func (h *htlcOutgoingContestResolver) Resolve() (ContractResolver, error) {
 	// Otherwise, we'll watch for two external signals to decide if we'll
 	// morph into another resolver, or fully resolve the contract.
 	//
-	// The output we'll be watching for is the *direct* spend from the HTLC
```

Comment:
Can this be added in a different PR to ease review? In this specific case, we only need to watch the mempool once the timeout is reached.